HW01/Ilya_Gribko/ilya_gribko__hw_01_t_01.rb

```diff
@@ -0,0 +1,48 @@
+# 1. If you look at the clock and the time is 3:15,
+# what is the angle between the hours and the minutes hands?
+# | If this task is too easy for you,
+# | try to implement a common solution for any time (3:15, 3:20, 4:30 ... etc )
+
+def angle_time
+  loop do
+    puts 'What time is it?(analog, e.g. 3:15, 3:20, 4:30 ... etc):'
+    time = gets.chomp
+
+    if time_check(time)
+      implement(time)
+      break
+    else
+      inv_format_message
```

Comment:
It would be cleaner to use the whole name
```suggestion
      invalid_format_message
```

Reply:
done